h2o-core/src/main/java/hex/ModelMetricsBinomialUplift.java

```diff
@@ -58,32 +59,64 @@ protected StringBuilder appendToStringMetrics(StringBuilder sb) {
 
     /**
      * Build a Binomial ModelMetrics object from target-class probabilities, from actual labels, and a given domain for both labels (and domain[1] is the target class)
-     * @param targetClassProbs A Vec containing target class probabilities
+     * @param predictedProbs A Vec containing predicted probabilities
      * @param actualLabels A Vec containing the actual labels (can be for fewer labels than what's in domain, since the predictions can be for a small subset of the data)
+     * @param treatment A Vec containing the treatment values               
+     * @param auucType Type of default AUUC
+     * @param nbins Number of bins to calculate AUUC (-1 means default value 1000, the number has to be higher than zero)  
      * @return ModelMetrics object
      */
-    static public ModelMetricsBinomialUplift make(Vec targetClassProbs, Vec actualLabels, Vec treatment, AUUC.AUUCType auucType, int nbins) {
-        return make(targetClassProbs, actualLabels, treatment, actualLabels.domain(), auucType, nbins);
+    static public ModelMetricsBinomialUplift make(Vec predictedProbs, Vec actualLabels, Vec treatment, AUUC.AUUCType auucType, int nbins) {
+        return make(predictedProbs, actualLabels, treatment, actualLabels.domain(), auucType, nbins, null);
     }
-    
+
     /**
      * Build a Binomial ModelMetrics object from target-class probabilities, from actual labels, and a given domain for both labels (and domain[1] is the target class)
-     * @param targetClassProbs A Vec containing target class probabilities
+     * @param predictedProbs A Vec containing predicted probabilities
+     * @param actualLabels A Vec containing the actual labels (can be for fewer labels than what's in domain, since the predictions can be for a small subset of the data)
+     * @param treatment A Vec containing the treatment values               
+     * @param auucType Type of default AUUC
+     * @param nbins Number of bins to calculate AUUC (-1 means default value 1000, the number has to be higher than zero)  
+     * @param customAuucThresholds custom threshold to calculate AUUC, if is not specified, the thresholds will be calculated from prediction vector             
+     * @return ModelMetrics object
+     */
+    static public ModelMetricsBinomialUplift make(Vec predictedProbs, Vec actualLabels, Vec treatment, AUUC.AUUCType auucType, int nbins, double[] customAuucThresholds) {
```

Comment:
This part could be in a different PR, but it is necessary for tests. But I can move it.